### PR TITLE
Project 2 - Mette Wickert

### PR DESCRIPTION
**Pull request by ettem**
My focus during the project was to finish the blue level goals but I also touched at least partly the red and black goals as well.

I had a hard time getting my connection to Github work, pushing the updates from VSC. Somehow I managed to delete my code in the clone I was working on, but I fixed it I think and hope everything is ok and works as it should so you can do a review.

I also struggled hard with flexbox and margins and paddings, hence I've added some classes and id's that probably could be merged together so the code will be less repetitive. Perhaps you have som advice concerning that. Also I like to get clarified concerning using id's for styling - should I stop using them and only use classes?

### CODE REVIEW

File: code/index.html [hunk 1]
```diff
@@ -1,22 +1,84 @@
...
+        <div class="logo">    
+          <img src="./KN_logo.svg" alt="logotype KnittyNews" />
+        </div>
+
+        <div class="nav">
+          <div class="nav-item"><a href="#">About</a></div>
+          <div class="nav-item"><a href="#">Archieve</a></div>
+          <div class="nav-item"><a href="#">Forum</a></div>
+          <div class="nav-item"><a href="#">Local</a></div>
+          <div class="nav-item"><a href="#">Contact</a></div>
```
Comment (HIPPIEKICK) [inline code/index.html:29]: Do you need these wrapping `div`s or would it be enough with just the links? If you decide to keep, give nested elements new lines:
```
<div>
  <a></a>
</div>
```

File: code/index.html [hunk 2]
```diff
@@ -1,22 +1,84 @@
...
+       </div>
+
+      </div>
     </header>
 
+
+  <div class="main">
+    
     <section class="big-news">
```
Comment (HIPPIEKICK) [inline code/index.html:38]: Some unnecessary blank lines here

File: code/index.html [hunk 3]
```diff
@@ -1,22 +1,84 @@
...
       <!-- The big news content -->
+        <div class="big-news-item1">
+          <a href="#"><img id="img-big" src="https://images.unsplash.com/photo-1499728603263-13726abce5fd?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1500&q=80" alt="image with phrase mindfulness" /></a>
+        </div>
+    
+        <div class="big-news-item2">
+          <a href="#"><h1 id="big-news-h1">Healthboost? <br>-positive effects of knitting</h1></a>
+          <p id="big-news-p">Lorem ipsum dolor sit amet consectetur adipisicing elit. A vel, similique inventore necessitatibus eaque dolores. Molestiae quae aliquam voluptates totam neque alias asperiores, numquam ab, quod tempore, voluptatum labore voluptas.</p>
```
Comment (HIPPIEKICK) [inline code/index.html:46]: Ids are mostly used for JavaScript, so to answer your question you could stick to style classes :) Read more in this weeks general feedback document :👉 [General Feedback - week 2](https://www.notion.so/technigo/Week-2-Project-news-site-bbe695289d154ff98d6a247cef873f07)

File: code/index.html [hunk 4]
```diff
@@ -1,22 +1,84 @@
...
+      </div>
     </header>
 
+
+  <div class="main">
+    
     <section class="big-news">
       <!-- The big news content -->
+        <div class="big-news-item1">
+          <a href="#"><img id="img-big" src="https://images.unsplash.com/photo-1499728603263-13726abce5fd?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1500&q=80" alt="image with phrase mindfulness" /></a>
```
Comment (HIPPIEKICK) [inline code/index.html:41]: New line

File: code/style.css [hunk 5]
```diff
@@ -1,4 +1,359 @@
...
+  background-position: center;
+  width: 100%;
+  height: auto;
+  }
+  
+  .flex-container {
+    display: flex;
+    flex-wrap: wrap;
+    flex-direction: row;
+  }
```
Comment (HIPPIEKICK) [inline code/style.css:20]: Check indentation of your CSS file, only the properties should be indented here, not the whole element :)

File: code/style.css [hunk 6]
```diff
@@ -1,4 +1,359 @@
...
+    opacity: 0.5;
+  }
+
+  a #third:hover{
+    opacity: 0.5;
+  }
+
+  a #fourth:hover{
+    opacity: 0.5;
+  }
```
Comment (HIPPIEKICK) [inline code/style.css:122]: Ids have to be unique, but for CSS it's more common to style by classes as mentioned, so you could give all these elements the same class and shorten this to one row

File: code/style.css [hunk 7]
```diff
@@ -1,4 +1,359 @@
...
+
+  #third {
+    object-fit: scale-down;
+    width: 100%;
+  }
+
+  #fourth {
+    object-fit: scale-down;
+    width: 100%;
+  }
```
Comment (HIPPIEKICK) [inline code/style.css:152]: Same here. If you were to style many elements in the same way though, you could do it with a comma:
```
.element-1, .element-2 {
  color: red;
}
```

File: code/style.css [hunk 8]
```diff
@@ -1,4 +1,359 @@
...
+    justify-content: flex-end;
+    width: 100%;
+    font-family: Verdana, sans-serif;
+    font-weight: bolder;
+    font-size: 22px;
+  }
+
+  .nav-item {
+    margin: 10px;
+    padding: 10px;
```
Comment (HIPPIEKICK) [inline code/style.css:40]: 10px is a bit small and hard to read 👀

File: code/style.css [hunk 9]
```diff
@@ -1,4 +1,359 @@
 html {
-  background: #0025ff;
-  color: #fff;
+  background: #ffffff;
+  color:grey;
 }
+
```
Comment (HIPPIEKICK) [inline code/style.css:5]: If you add:
```
body {
  margin: 0;
}
```
your image header would go all the way to the edges :)

**Review by HIPPIEKICK — CHANGES_REQUESTED** [review]
Good job with this project Mette! I really like the topic and the colors 🤩 As you mentioned, your code is a little repetitive but I gave you some comments on how you can think about that. I would like to ask you take a look at your formatting, so new line (and indentation) for each nested HTML element. Also, take a look at the indentation of your CSS file. For further reference - [code guidelines](https://www.notion.so/technigo/Guidelines-for-how-to-write-good-code-69ffb59eac4e4358a7c5752efb873c4e). Change this and you're approved :)

We’ve also put together some general feedback that applies to many projects this first week. Check it out and take the tips with you to upcoming projects! 👉 [General Feedback - week 2](https://www.notion.so/technigo/Week-2-Project-news-site-bbe695289d154ff98d6a247cef873f07)

File: code/style.css [hunk 10]
```diff
@@ -1,4 +1,359 @@
...
+  .iheart {
+    color: darkgrey;
+    font-size: 20px;
+    animation: colorswap 5s infinite alternate;
+  }
+
+  @keyframes colorswap {
+    from {color: darkgrey;}
+    to {color: red;}
+  }
```
Comment (HIPPIEKICK) [inline code/style.css:171]: 🤩

**Comment by ettem** [comment]
Thank you Matilda for your good and constructive review! I have updated and pushed the changes you requested, I hope I interpreted them correctly :) Also, Github is not my friend yet so I hope I've done this right so you can see the changes.

File: code/index.html (hunk 4 above)
Comment (HIPPIEKICK) [reply code/index.html:49]: <a href="#">
  image with phrase mindfulness
</a>

File: code/style.css [hunk 11]
```diff
@@ -11,347 +15,316 @@ html {
...
-  .big-news {
-    display: flex;
-    flex-wrap: wrap;
-    flex-direction: row;
-    justify-content: center;  
-    background-color: #E2F3EB; 
-  }
+background-color: #C5E3D5;
+padding: 10px 0 10px 0;
+font-family: verdana, sans-serif;
```
Comment (HIPPIEKICK) [inline code/style.css:51]: These two lines too 😇

**Comment by HIPPIEKICK** [comment]
> Thank you Matilda for your good and constructive review! I have updated and pushed the changes you requested, I hope I interpreted them correctly :) Also, Github is not my friend yet so I hope I've done this right so you can see the changes.

Yes, you've got the hang of it 💯  Approving!

**Review by HIPPIEKICK — APPROVED** [review]
